Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift

```diff
@@ -240,29 +254,35 @@ class PhotonActionSheet: UIViewController, Themeable {
     }
 
     private func applyBackgroundBlur() {
-        guard let sceneDelegate = UIApplication.shared.connectedScenes.first?.delegate as? SceneDelegate,
-              let screenshot = sceneDelegate.window?.screenshot() else { return }
+        guard let screenshot = windowScreenshot else { return }
 
+        let blurType: BlurType = UIAccessibility.isReduceTransparencyEnabled ? NOBLUR : BOXFILTER
         let blurredImage = screenshot.applyBlur(withRadius: 5,
-                                                blurType: BOXFILTER,
+                                                blurType: blurType,
                                                 tintColor: UIColor.black.withAlphaComponent(0.2),
```

Comment:
Wondering if the tintColor of the blurred image should be themeable?

Reply:
I don't think it should and it was also approved by design in this [PR](https://github.com/mozilla-mobile/firefox-ios/pull/12657).